lib/envars.ml

```diff
@@ -12,7 +12,13 @@ open Util
 
 (** {1 Helper functions} *)
 
-let getenv_else s dft = try Sys.getenv s with Not_found -> dft ()
+let getenv_else s dft =
+  try
+    let v = Sys.getenv s in
+    if String.equal v ""
+    then dft ()
+    else v
+  with Not_found -> dft ()
```

Comment:
Why is this needed?

Reply:
I was doing as suggested here https://github.com/coq/coq/issues/8731#issuecomment-429842201

Reply:
So indeed it makes Coq to act with the dummy setting of COQLIB in the makefile as if it was not set. Pretty hacky tho.

Reply:
Then why is the other change needed?

Reply:
Likely it broke some other stuff? I wrote this patch long time ago so trying to remember...

Reply:
I think what happened is that the first hunk was suggested first, but it's insufficient as COQLIB still gets overridden in the makefile so we would get the default even when the user set it in the environment.
Then the second hunk was written and made the first one not needed.